packages/liquidator/index.js

```diff
@@ -210,6 +222,7 @@ async function Poll(callback) {
     // match the order of parameters defined in the`run` function.
     const executionParameters = {
       empAddress: process.env.EMP_ADDRESS,
+      oneSplitAddress: ONE_SPLIT_ADDRESS || process.env.ONE_SPLIT_ADDRESS,
```

Comment:
All other parameters in this object have some kind of comment. could you please add one here?